subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileLineEndingSensitivityIntegrationTest.groovy

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.tasks.compile
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.util.TextUtil
+
+class JavaCompileLineEndingSensitivityIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
You can implement `DirectoryBuildCacheFixture`, which will take care of setting the build cache directory.

Reply:
Ah, thanks for making me aware of that.  Done.